.drone.windows.jsonnet

```diff
@@ -0,0 +1,9 @@
+local pipeline = import 'pipeline.libsonnet';
+local name = 'drone-webhook';
+
+[
+  pipeline.test('windows', 'amd64', '1803'),
```

Comment:
IMHO we could skip test and vet on windows

Reply:
Eh its not a big deal to test the thing on Windows as well. There's always the chance for some platform specific tests.